privacyidea/api/smtpserver.py

```diff
@@ -72,10 +72,11 @@ def create(identifier=None):
     tls = getParam(param, "tls", default=False)
     tls = tls in ["True", True, "true", "1"]
     description = getParam(param, "description", default="")
+    timeout = int(getParam(param, "timeout", default=10))
```

Comment:
We still have a similar problem :/
Steps: Have SMTP Server without timeout defined in master, migrate database, edit the SMTP server, do not touch the timeout field, save:
```[2018-02-09 13:22:45,949][4072][140416845743296][ERROR][privacyidea.app:1423] Exception on /smtpserver/local [POST]
Traceback (most recent call last):
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/fred/privacyidea/privacyidea/venv/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/fred/privacyidea/privacyidea/privacyidea/api/lib/prepolicy.py", line 117, in policy_wrapper
    return wrapped_function(*args, **kwds)
  File "/home/fred/privacyidea/privacyidea/privacyidea/lib/log.py", line 154, in log_wrapper
    return func(*args, **kwds)
  File "/home/fred/privacyidea/privacyidea/privacyidea/api/smtpserver.py", line 75, in create
    timeout = int(getParam(param, "timeout", default=10))
TypeError: int() argument must be a string or a number, not 'NoneType'
```
Maybe we should also use ``or 10`` here?